otterdog/eclipse-orbit.jsonnet

```diff
@@ -58,5 +58,13 @@ orgs.newOrg('eclipse-orbit') {
       has_wiki: false,
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('orbit-legacy') {
```

Comment:
minor nitpick: if you add the repos in sorted order by repo-name, the canonical diff would show no differences.

This would allow you to spot redundant settings in your config easier, as changes just due to sorting would not be present.